package.json

```diff
@@ -180,6 +180,7 @@
     "slate-soft-break": "^0.4.0",
     "slug": "^0.9.1",
     "toml-j0.4": "^1.1.1",
+    "tomlify-j0.4": "^3.0.0-alpha.0",
```

Comment:
3.0.0 is still officially an alpha, but it should stay stable (see https://github.com/jakwings/tomlify-j0.4/issues/4).